haystack/document_stores/filter_utils.py

```diff
@@ -1,12 +1,11 @@
 from typing import Union, List, Dict, Optional, Tuple
 from abc import ABC, abstractmethod
 from collections import defaultdict
-from functools import reduce
 
 from sqlalchemy.sql import select
 from sqlalchemy import and_, or_
 
-from haystack.document_stores import utils
+from haystack.document_stores.utils import convert_date_to_rfc3339
```

Comment:
Test currently fail due to this line. I already faces this problem when I added the `filter_utils.py` file. The problem was about a circular import. That's why I needed to import the whole module and access the `convert_date_to_rfc3339` via `utils.convert_date_to_rfc3339`.

Reply:
Tha is happening because the function `elasticsearch_index_to_document_store` uses filters. However, by the name, I think I could put it in the `elasticsearch.py` file and solve the problem that way. Sounds good to you? I don't see why the function has been put in the utils to begin with.

Reply:
We decided to not put it in `elasticsearch.py`, as this works with any DocumentStore. See also this comment: https://github.com/deepset-ai/haystack/pull/2229#pullrequestreview-889928550

Reply:
I see! Then, to avoid the circular import, we can create a separate module for these two functions. How could it be named? `es_to_docstore.py`? `es_exporter.py`?

Reply:
Both are fine to me, I would lean a bit more towards `es_converter.py`, seems to be a bit more explicit to me.